enkit/outputs/commands.go

```diff
@@ -375,28 +357,22 @@ func NewShutdown(root *Root) *Shutdown {
 }
 
 func (c *Shutdown) Run(cmd *cobra.Command, args []string) error {
-	bbOpts := bbexec.NewClientOptions(
-		c.root.Log,
-		"", // Buildbarn remote host/port does not matter
-		0,
-		c.root.OutputsRoot,
-	)
-	var errs []error
-	// MaybeStartClient is used here to bind a client handle to an existing process, so that we can kill it. It may start a process that will be then killed quickly, which is acceptable but not ideal.
-	bbClient, err := bbexec.MaybeStartClient(bbOpts, 5*time.Second)
-	if err != nil {
-		errs = append(errs, err)
-	}
-	if bbClient != nil {
-		if err := bbClient.Shutdown(); err != nil {
-			errs = append(errs, fmt.Errorf("error maybe? killing the process of existing bb_clientd %v", err))
-		}
-	}
-	if err := os.RemoveAll(c.root.OutputsRoot); err != nil {
-		errs = append(errs, err)
-		c.root.Log.Errorf("error removing output root %s %v", c.root.OutputsRoot, err)
-		return multierror.New(errs)
-	}
-	fmt.Printf("Successfully deleted output root at %s \n", c.root.OutputsRoot)
-	return nil
+        dir := DefaultOutputsRoot
```

Comment:
I used ChatGPT for this function so might need some extra scrutiny

Reply:
I don't quite get it. The below looks like it's going to ignore the files in the top directory.

Reply:
The top directory is `enf_mounts/buildbarn/scratch`: a fuse filesystem managed by `bb_clientd` - used for syncing files from CAS to a writeable location that is garbage collected between `bb_clientd` restarts.

The `enkit outputs mount` command will create a subdirectory here under `<invocation-id>` with all of the relevant files from that particular bazel invocation.

The purpose of the `enkit outputs shutdown` subcommand is to delete all of the invocation dirs that were created with `enkit outputs mount`.